db/migrate/20250719235626_create_tasks.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTasks < ActiveRecord::Migration[8.0]
+  def change
+    create_table :tasks do |t|
+      t.string :name
```

Comment:
Are we correctly adding the not-null constraints to the database to match the validations?